src/v/debug_bundle/debug_bundle_service.cc

```diff
@@ -200,11 +219,13 @@ ss::future<> service::start() {
         co_return;
     }
 
-    try {
-        lg.trace("Creating {}", _debug_bundle_dir);
-        co_await ss::recursive_touch_directory(_debug_bundle_dir.native());
-    } catch (const std::exception& e) {
-        throw std::system_error(error_code::internal_error, e.what());
+    if (!co_await ss::file_exists(_debug_bundle_dir.native())) {
+        try {
+            lg.trace("Creating {}", _debug_bundle_dir);
+            co_await ss::recursive_touch_directory(_debug_bundle_dir.native());
+        } catch (const std::exception& e) {
+            throw std::system_error(error_code::internal_error, e.what());
+        }
```

Comment:
nitpick: this logic duplicated from `initiate_rpk_debug_bundle_collection` w/ slightly different logging and error reporting